manim/mobject/coordinate_systems.py

```diff
@@ -1355,6 +1355,35 @@ class NumberPlane(Axes):
 
     .. note:: If :attr:`x_length` or :attr:`y_length` are not defined, the plane automatically adjusts its lengths based
         on the :attr:`x_range` and :attr:`y_range` values to set the unit_size to 1.
+
+    Examples
+    --------
+
+    .. manim:: NumberPlaneExample
+        :ref_classes: NumberPlane
```

Comment:
```suggestion
```

Reply:
I think this reference to the own class name is not needed.
Thanks a lot for the contribution!